card/mod/02_basic_types/spec/set/all/base_spec.rb

```diff
@@ -7,6 +7,8 @@
     it("name"    ) { expect(render_card(:name)).to      eq('Tempo Rary') }
     it("key"     ) { expect(render_card(:key)).to       eq('tempo_rary') }
     it("linkname") { expect(render_card(:linkname)).to  eq('Tempo_Rary') }
+    it("plural")   { expect(render_card(:plural)).to    eq('Tempo Raries') }
+    it("singular")   { expect(render_card(:singular)).to    eq('Tempo Rary') }
```

Comment:
Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.